src/sentry/rules/conditions/active_release.py

```diff
@@ -14,7 +14,24 @@ class ActiveReleaseEventCondition(EventCondition):
     id = "sentry.rules.conditions.active_release.ActiveReleaseEventCondition"
     label = "A new issue is created within an active release (1 hour of deployment)"
 
-    def is_in_active_release(self, event: Event) -> bool:
+    def passes(self, event: Event, state: EventState) -> bool:
+        if self.rule and self.rule.environment_id is None:
+            return (state.is_new or state.is_regression) and self.is_in_active_release(event)
+        else:
+            return (
+                state.is_new_group_environment or state.is_regression
+            ) and self.is_in_active_release(event)
+
+    @staticmethod
+    def latest_release(event: Event) -> Optional[Release]:
+        return Release.objects.filter(
+            organization_id=event.project.organization_id,
+            projects__id=event.project_id,
+            version=event.release,
+        ).first()
+
+    @staticmethod
+    def is_in_active_release(event: Event) -> bool:
```

Comment:
Besides moving these util methods to another file, is there a better way to do this to avoid circular imports?

Reply:
Another way is to move `from sentry.rules.conditions.active_release import latest_release` into `__init__` where it's called. This is probably nicer though.